uimaj-core/pom.xml

```diff
@@ -260,4 +260,36 @@
       </plugin>
     </plugins>
   </build>
+  <profiles>
+    <!-- ************ Java 9 enablement ************** -->
```

Comment:
I wonder if we could name this java 9 or 10 enablement?

Reply:
`Java 9+` ? ; )